### ISSUE TITLE
Screen readers (NVDA/JAWS) are not narrating the Saved information upon invoking 'Save' button in the page

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#588]
Impact on a Person with Disability (PWD):
Users who rely on the screen reader will be impacted if Screen readers (NVDA/JAWS) are not narrating the Saved information upon invoking 'Save' button in the page. As a result, end-users will not get the information about the action performed by activating the button so the end-users will get confused, and this may lead to disorientation.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Screen Readers: NVDA (2023.3), JAWS (2022.2110.6)

Pre-requisites:
Turn on NVDA/JAWS

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to '...' ellipsis control and invoke it.
Step 4: Navigate to Create new retrospective button and invoke it.
Step 5: Create new retrospective Dialog gets appear.
Step 6: Fill all required fields and invoke Save button.
Step 7: Verify whether Screen readers (NVDA/JAWS) are narrating the Saved information upon invoking 'Save' button in the page or not.

Actual Result: 
Screen readers (NVDA/JAWS) are not narrating the Saved information upon invoking 'Save' button in the page.
Current Observation: Saved message is not appearing after invoking 'Save' button and NVDA is not narrating any Saved information.

Expected Result: 
Screen readers (NVDA/JAWS) should narrate the Saved information upon invoking 'Save' button in the page.
Ex: Saved message should appear after invoking 'Save' button and NVDA should announce that saved message information upon invoking 'Save' button.

MAS Reference:
[MAS 4.1.3 – Status Messages](https://aka.ms/MAS4.1.3)

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713373
